lib/bedrock/index.js

```diff
@@ -15,10 +15,10 @@ module.exports = (data) => {
 
     writeItemStates () {
       const itemstates = []
-      for (const item of Object.values(data.items)) {
+      for (const item of data.itemsArray) {
         // Custom items with different namespaces can also be in the palette
         // There might be a better way to test for this
-        const name = /(\w+?):(\w+)/.test(item.name) ? item.name : `minecraft:${item.name}`
+        const name = /(.+?):(.+)/.test(item.name) ? item.name : `minecraft:${item.name}`
```

Comment:
Do not use regex 
```js
let [ns, name] = item.name.split(':'); 
if (!ns) ns = 'minecraft'
name = ns + ':' + name
```

Since we are not supporting any non-minecraft namespaces for the time being, adding a startsWith(minecraft) and prefixing is good also.